packages/cli/src/commands/deploy.ts

```diff
@@ -147,7 +149,7 @@ const deployPlan = async (
       workspace,
       actionPlan,
       (item: PlanItem, step: ItemStatus, details?: string) =>
-        updateAction(item, step, details),
+        updateAction(item, step, details), checkOnly,
```

Comment:
nit: no reason to hide this parameter
```suggestion
        updateAction(item, step, details),
        checkOnly,
```